src/components/CourseCard/__test__/CourseCard.test.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import CourseCard from '../CourseCard';
+import { Provider } from 'react-redux';
+import store from '../../../store/index';
+import { BrowserRouter } from 'react-router-dom';
+
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
import this in `jest.setup` file instead https://jestjs.io/ru/docs/configuration#setupfilesafterenv-array